cli/src/config/templates.toml

```diff
@@ -91,12 +91,9 @@ if(root,
   )
 )
 '''
-builtin_log_compact = '''
-if(root,
-  format_root_commit(self),
-  label(if(current_working_copy, "working_copy"),
-    concat(
-      separate(" ",
+
+builtin_log_compact_commit_header = '''
```

Comment:
nit: `format_...(commit)` for naming consistency and hide it from the list of builtin aliases.